code/headers/ball_c.hpp

```diff
@@ -0,0 +1,25 @@
+#include <stdint.h>
```

Comment:
This file lacks *#pragma once*. Is this intended?

Reply:
forgot, thanks!

Reply:
repaired!